### PR TITLE
fix(args): check if required args like src, dest are provided

### LINKED ISSUE #5: [bug] checking that src and dest args are provided is not done.
Currently if we don't pass **dest** this error comes
image

(error repeated 100 times for 100 svg files)

and if we don't pass **src** this error comes

image

so it should be checked if user provided src and dest args or not.

### PR DESCRIPTION
**Pull request by itaditya**
fix #5

### CODE REVIEW

**Comment by itaditya** [comment]
Here's how it would look

image

File: src/cli.js [hunk 1]
```diff
@@ -77,6 +77,18 @@ async function generateFiles() {
 
 (async () => {
   try {
+    if (!src) {
```
Comment (bharathnayak03) [inline src/cli.js:80]: we can do it using, yargs itself

**Review by bharathnayak03 — CHANGES_REQUESTED** [review]

**Comment by bharathnayak03** [comment]
closing this . as the change was done using yargs options

File: src/cli.js (hunk 1 above)
Comment (itaditya) [reply src/cli.js:80]: great!

**Pull request closed without merging**